### PR TITLE
Merge similar pancake recipes

### PR DESCRIPTION
**Pull request by DavGoeck**

### CODE REVIEW

File: src/pancake.md [hunk 1]
```diff
@@ -1,25 +1,33 @@
...
+- Syrup
+- Cinnamon and sugar
 
-BeFe
+## Contribution
+- BeFe (original)
+- Lxgr (gluten-free) - [github](https://github.com/lxgr-linux)
 
-;tags: quick breakfast sweet pancake cheesefare
+;tags: quick basic breakfast sweet pancake cheesefare
```
Comment (hallzy) [inline src/pancake.md:33]:
```suggestion
;tags: quick breakfast sweet pancake cheesefare
```
Not sure if basic should be added. It's really meant for recipes that are not meant to be done alone as a dish, but pancakes on their own (except for the addition of syrup or something) can pretty much be their own thing.

**Review by hallzy — APPROVED** [review]

File: src/pancake.md (hunk 1 above)
Comment (hallzy) [reply src/pancake.md:33]: Also, 6 tags is too many

**Comment by hallzy** [comment]
Failed check can be ignored. The newer version of the PR script I have in a PR (#462) wouldn't fail. it is just because a file was removed in this PR and the current PR script doesn't account for that.

**Review by DGKSK8LIFE — APPROVED** [review]

**Comment by DavGoeck** [comment]
I think that all pancake recipes should be merged into a single one. The differences are marginal and can be recorded as variations similar to the egg recipe. The only thing holding me back from doing it is that I never made pancakes and cannot judge which one is the most standard one. I also don't want to screw up the recipes when rescaling the different recipes to a common denominator.

**Comment by siggines** [comment]
This should not be pulled, it is too similiar to https://based.cooking/pancake

**Pull request merged**